app/index.js

```diff
@@ -143,6 +145,11 @@ const listen = async (port = null) => {
   for (const {methods, path} of routes) {
     logger.info(`Registered route: (${Object.keys(methods).sort().join('|')}) ${path}`);
   }
+
+  // bull-mq setup
+  const queues = setupQueues();
+  setupBullBoard({app, queues});
+
```

Comment:
can we add a logger line here indicating that bull board/ queue setup is complete